modules/vstudio/vs2010.lua

```diff
@@ -80,6 +80,19 @@
 						p.touch(prj, ".vcxitems")
 					end
 				end
+
+			elseif prj.kind == p.PACKAGING then
+
+				if project.iscpp(prj) then
```

Comment:
In my project I've not had to set the system for the packaging project. If I add this check now it breaks the project and is unable to load it. It's not a big issue but also not trivial to diagnose if one forgets. Perhaps I should add the check and an error message? How would I go about doing that?

Reply:
Hmm, this is now outside the scope of this PR, but if you try to use `kind "Packaging"` on Windows, it just generates a `.vcxproj`. I assume with your change it might generate a `.androidproj` instead? That's not necessarily worse, but we should probably emit an error (in a future PR) - that can be done using `p.error`.

Reply:
That's interesting, the code I have for creating a packaging project is

```lua
project (AndroidProject)
	removeplatforms("*")
	platforms { PlatformAndroidARM, PlatformAndroidARM64 }
	kind("Packaging") -- This type of project builds the apk
	links (UnitTestProject) -- Android needs to link to the main project which was built as a dynamic library
	androidapplibname(UnitTestProject)
	files
	{
		unitTestDir.."/android/AndroidManifest.xml",
		unitTestDir.."/android/build.xml",
		unitTestDir.."/android/project.properties",
		unitTestDir.."/android/res/values/strings.xml",
	}
```

and it creates the file correctly. I can add the system and the error when I get a minute

Reply:
> I can add the system and the error when I get a minute

I'm happy for this to be done in a follow up PR if you'd prefer.

Reply:
I have done that now, it was easy enough. It's easy enough to update my script and makes sense